src/aead/chacha20_poly1305.rs

```diff
@@ -278,6 +279,67 @@ mod tests {
                     (CRYPTO_POLY1305_STATE_LEN + 255) / 256);
     }
 
+    #[test]
+    pub fn chacha20_tests(){
```

Comment:
Make sure that the spacing matches the existing conventions throughout your patch, e.g. `s/(){/() {/`.